pkg/client/listers/admission/internalversion/admittancereview.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This file was automatically generated by lister-gen
+
+package internalversion
+
+import (
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/tools/cache"
+	admission "k8s.io/kubernetes/pkg/apis/admission"
+)
+
+// AdmittanceReviewLister helps list AdmittanceReviews.
+type AdmittanceReviewLister interface {
```

Comment:
Are the listers a left-over from when we had ObjectMeta? If I read https://github.com/kubernetes/kubernetes/blob/master/cmd/libs/go2idl/lister-gen/generators/lister.go#L82 correctly, these shouldn't be generated.